src/svgutils/transform.py

```diff
@@ -141,14 +141,18 @@ class TextElement(FigureElement):
     Corresponds to SVG ``<text>`` tag."""
     def __init__(self, x, y, text, size=8, font="Verdana",
                  weight="normal", letterspacing=0, anchor='start',
-                 color='black'):
-        txt = etree.Element(SVG+"text", {"x": str(x), "y": str(y),
-                                         "font-size": str(size),
-                                         "font-family": font,
-                                         "font-weight": weight,
-                                         "letter-spacing": str(letterspacing),
-                                         "text-anchor": str(anchor),
-                                         "fill": str(color)})
+                 color='black', style="normal", opts={}):
```

Comment:
be careful not to put mutable objects (dictionaries, lists, etc) as default arguments, as this may change the default, if someone changes the object.